How to offboard an employee (#696)
flowforge/website: src/handbook/peopleops/hiring.md

### Involuntary Resignation

When a team member is being let go, it is usually due to either performance issues or a change in the company's needs that no longer justifies their role. If the decision is due to performance issues, we will have already provided feedback and given the team member an opportunity to improve. Unfortunately, if the issues cannot be resolved, termination of employment may be necessary. In most cases, we will ask the team member to stop working immediately. The final payment will be made according to the terms of their employment contract.

### Communicating Departures

When a team member voluntarily departs, we may ask if they would like to share their plans with the team. If the departure is involuntary,  reasons of the departure will be at the sole discretion of the FlowForge management.

## Offboarding process

We will send an email to the departing employee covering the following points:

1. Final payment

> 1. Vested share options

Reviewer: ```suggestion
```

that's no longer available is a waste of their time and a bad experience with
FlowForge and hurts our reputation.

## Offboarding
The purpose of the offboarding section in the handbook is to provide a clear and structured framework for employees at FlowForge to follow during the offboarding process. The offboarding process is managed by the People Ops Manager in close collaboration with the CEO, Reporting Managers, and Deel for Payroll at FlowForge.

When a team member or contractor decides to leave FlowForge, they will receive a detailed email before their final date of employment. The email will include information regarding the impact on benefits coverage, stocks, final pay, and the process for returning any company assets.

### Voluntary Resignation

> If a team member decides to leave FlowForge, we request them to provide a 30-day notice period by default, unless specified otherwise in their employment contract on Deel. During this period, we expect the team member to continue working to facilitate a smooth handover and enable us to find a suitable replacement. Please note that we generally expect the team member to work for the entire notice period.

Reviewer: Let's keep the contracts the contracts, not in the handbook to avoid confusion.

```suggestion
If a team member decides to leave FlowForge they're expected to continue working to facilitate a smooth handover and enable us to find a suitable replacement. Please note that we generally expect the team member to work for the entire notice period.
```

If you are considering resigning from your position at FlowForge, we highly recommend discussing your reasons with your manager or the People Ops manager. We are always open to improving and making changes that benefit our team members. While we understand that sometimes leaving is the best option, we encourage you to explore all possible solutions before making a final decision.

Once you have discussed your concerns with your manager and the People Ops team and determined that resigning from FlowForge is the only option, please notify both your manager and the People Ops team of your intention to resign via email. Afterward, we will commence a discussion to establish the steps required to ensure a seamless handover.

### Involuntary Resignation

> When a team member is being let go, it is usually due to either performance issues or a change in the company's needs that no longer justifies their role. If the decision is due to performance issues, we will have already provided feedback and given the team member an opportunity to improve. Unfortunately, if the issues cannot be resolved, termination of employment may be necessary.In most cases, we will ask the team member to stop working immediately. The final payment will be made according to the terms of their employment contract.

Reviewer: ```suggestion
When a team member is being let go, it is usually due to either performance issues or a change in the company's needs that no longer justifies their role. If the decision is due to performance issues, we will have already provided feedback and given the team member an opportunity to improve. Unfortunately, if the issues cannot be resolved, termination of employment may be necessary. In most cases, we will ask the team member to stop working immediately. The final payment will be made according to the terms of their employment contract.
```

### Communicating Departures

> When a team member voluntarily departs, we may ask if they would like to share their plans with the team. If the departure is involuntary, we will strive to maintain transparency regarding the reasons for the departure, while still respecting the individual's privacy.

Reviewer: While I agree, it's a bit vague. I don't think we should tell anyone it's performance related, or anything of the kind.

Author: I have made changes to this section.

Please note that in some instances, FlowForge may be unable to provide additional context surrounding an employee's departure.

## Offboarding process

We will send an email to the departing employee covering the following points:

> Final payment
> Vested share options
> Return of company property
> Reimbursement of business expenses

Reviewer: ```suggestion
1. Final payment
1. Vested share options
1. Return of company property
1. Reimbursement of business expenses
```

At FlowForge, we conduct Exit Interviews for employees who choose to leave the company voluntarily. This provides us with valuable feedback on their experience, reasons for leaving, and areas for improvement. The People Ops manager schedules these interviews during the employee's last week to ensure they have an opportunity to share their insights. Our aim is to use this feedback to enhance our workplace culture and practices.

### Offboarding Issue

> The People Ops manager will create a new offboarding Issue for each leaver using the template maintained on GitHub, which can be viewed https://github.com/flowforge/admin/issues/166

Reviewer: ```suggestion
The People Ops manager will create a new off boarding issue for each instance of a team member leaving. The template maintained on GitHub, create a new issue though [this link](https://github.com/flowforge/admin/issues/new/choose).
```

> ### Final Pay
> 
> Final pay for employees leaving FlowForge will be determined based on their length of service and the reason for leaving. If the offboarding is voluntary, the employee will be paid up until their last day and any unused vacation pay will be paid out based on assuming the employee would have taken 25 days of vacation (as we offer unlimited PTO).
> 
> If offboarding is involuntary due to performance or business changes, severance pay will be provided at the sole discretion of FlowForge management.

Reviewer: Much of this section is based on what Deel dictates and where the employee is located, what about:

```suggestion
### Final Pay

Final pay consists of the statutory minimal payment dictated by our payroll provider Deel, and an optional extra severance package. Such a package might be dependent on on business requirements, standing of the employee, among other factors.
```

> ## Share options vested
> 
> We will confirm the number of vested share options and provide information on how to exercise them, if applicable.

Reviewer: The peopleops manager informs Cooley and Formation Financial, these update Carta. Carta includes all required details. We can inform them in an email to check carta, but the legal issues if we get this part wrong are too big to provide this service I think. Let's just depend on Carta.

```suggestion
## Share options vested

We will confirm the number of vested share options and provide information on how to exercise them, if applicable.
```

Author: I have removed this section from the handbook for now.
